File: timetrials/src/main/java/fzero/timetrials/repository/TimetrialRepository.java
package fzero.timetrials.repository;

import fzero.timetrials.game.TimeTrialRun;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface TimetrialRepository extends JpaRepository<TimeTrialRun, Long> {

}

File: FZero99-TT/src/main/kotlin/fZero/fZero99TT/repository/TimetrialRepository.kt
package fZero.fZero99TT.repository

import fZero.fZero99TT.game.FZeroTrack
import fZero.fZero99TT.game.TimeTrialRun
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface TimetrialRepository : JpaRepository<TimeTrialRun, Long> {
    fun findByTrack(track: FZeroTrack): List<TimeTrialRun>
}
